### ISSUE TITLE
Config Flow Could Not Be Loaded - "Invalid handler specified"

### ISSUE THREAD

**Issue by mstmagic** [bjhiltbrand/drone_mobile_home_assistant#36]
Hello,

I'm experiencing an issue when trying to install the DroneMobile integration in Home Assistant. After placing the drone_mobile folder into the custom_components directory and restarting Home Assistant, I attempted to add the integration via the UI but encountered an error.

Steps to Reproduce:
Downloaded the latest code from [drone_mobile_home_assistant](https://github.com/bjhiltbrand/drone_mobile_home_assistant).
Placed the drone_mobile folder into the custom_components directory of my Home Assistant configuration.
Restarted Home Assistant to recognize the new custom component.
Navigated to Settings > Devices & Services > Add Integration.
Selected DroneMobile from the list.
Expected Behavior:

The integration's configuration flow starts, allowing me to input my credentials and set up the DroneMobile integration.

Actual Behavior:
An error message appears stating:
Error: Config flow could not be loaded: {"message":"Invalid handler specified"}

Relevant Logs:
Here is a warning from the logs, though it might be unrelated:
2024-12-01 16:01:16.206 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration drone_mobile which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant

Home Assistant Version: 2024.11.3
Docker Image: lscr.io/linuxserver/homeassistant:latest
Installation Method: Home Assistant OS / Supervised / Container / Core (please specify)
Python Version: 3.12.4

Troubleshooting Attempts:
Verified that the directory structure is correct, with all necessary files (__init__.py, manifest.json, config_flow.py, etc.) inside the drone_mobile folder.
Checked the manifest.json file to ensure "config_flow": true is present and that there are no syntax errors.
Reviewed the config_flow.py file to confirm it exists and is properly implemented.
Looked at Home Assistant logs for any detailed error messages but only found the warning mentioned above.

Possible Cause:
It seems the integration might be incompatible with the current version of Home Assistant due to changes in the configuration flow handling or deprecated methods.

Request:
Could you please look into this issue? Any guidance on how to resolve this error would be greatly appreciated. If there's a need to update the integration to be compatible with the latest version of Home Assistant, I'd be happy to assist with testing.

Thank you for your time and effort in maintaining this integration!

**Comment by 2rememberyou** [comment]
Yes unfortunately 2025.1 broke the integration for me. I think some things still work but definitely not locks. This is my favorite integration. I truly hope these gentelman find the time to address this.

**Comment by pushypush** [comment]
Agreed!

**Comment by bjhiltbrand** [comment]
I pushed the Pull Request that fixes this into the main branch. This should be working again.
